src/Components/Shared/Inputs/TextArea.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './inputs.module.css';
+
+const TextArea = ({ value, onChangeArea, placeholder }) => {
```

Comment:
should be great to have props for "name" and "label"

Reply:
👍